components/details/DetailsCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useRouter } from "next/router";
+import moment from "moment/moment";
+import ItemCategoryIcon from "../ItemCategoryIcon";
+import Link from "next/link";
+import styled from "styled-components";
+import { DetailsCardStyled, StyledLink, StyledShopLink } from "./DetailsStyled";
+
+export default function DetailsCard({ items }) {
+  function findMe(item) {
```

Comment:
```suggestion
  function findMe(item) {
```

is `Me` really the right term?